### DISCUSSION TITLE
Support  application specific setting on COSMIC desktop environment

### DISCUSSION THREAD

**Discussion by Eason0210** [xremap/xremap#635] [Ideas]
Hi, 

 [COSMIC](https://en.wikipedia.org/wiki/COSMIC_(desktop_environment)) is a desktop environment developed in the Rust. Recenly is in alpha4 stage.

It can be install on Arch Linux with pacman:
```
sudo pacman -S cosmic 
```

Is there plan to support this new DE?

Rencently, xremap can not support the application specific setting for COSMIC. I got the following  message:

```
application-client: none (supported: false)
```

**Comment by k0kubun** [comment]
I don't use COSMIC, so I can't work on that myself. Somebody using COSMIC needs to file a PR to support it. I would merge that PR and make them a maintainer for the COSMIC support.

**Comment by hpccc53** [comment]
This is now possible from version 0.14.7

**Reply by Eason0210** [reply]
Thanks for adding support for Cosmic, it works well on Cosmic now.
